app/src/main/java/at/roteskreuz/stopcorona/model/repositories/InfectionMessengerRepository.kt

```diff
@@ -231,7 +302,9 @@ class InfectionMessengerRepositoryImpl(
         }
     }
 
-    private suspend fun fetchDailyBatchesDiagnosisKeys(dailyBatches: List<ApiDiagnosisKeysBatch>): List<DbDailyBatchPart> {
+    private suspend fun fetchDailyBatchesDiagnosisKeys(dailyBatches: List<ApiDiagnosisKeysBatch>
```

Comment:
not sure if it's only a wrong formatting here in the reader, please make sure to have either 
```
private suspend fun fetchDailyBatchesDiagnosisKeys(dailyBatches: List<ApiDiagnosisKeysBatch>) : List<DbDailyBatchPart> {
```
(in one line)

or
```
private suspend fun fetchDailyBatchesDiagnosisKeys(
     dailyBatches: List<ApiDiagnosisKeysBatch>
) : List<DbDailyBatchPart> {
```

our codestyle helps you with that

Reply:
done